core/templates/dev/head/domain/collection/CollectionRightsBackendApiService.js

```diff
@@ -41,7 +41,7 @@ oppia.factory('CollectionRightsBackendApiService', [
           }
         }, function(error) {
           if (errorCallback) {
-            errorCallback(error);
+            errorCallback(error.data);
```

Comment:
Here and elsewhere: maybe call this errorResponse instead of error? We should be consistent.